pkg/crcontrollers/lvg/lvgcontroller.go

```diff
@@ -157,6 +157,7 @@ func (c *Controller) handlerLVGCreation(lvg *lvgcrd.LVG) (ctrl.Result, error) {
 	if locations, err = c.createSystemLVG(lvg); err != nil {
 		ll.Errorf("Unable to create system LVG: %v", err)
 		newStatus = apiV1.Failed
+		lvg.Spec.Health = apiV1.HealthBad
```

Comment:
This looks confusing. Do we use `Health` field for logical health as well? I also don't see where we ignore such LVGs for allocation...

Reply:
We update LVG status to `Failed` in case of drive bad health. If LVG has `Failed` status. CSI resets its AC. That was implemented here https://github.com/dell/csi-baremetal/pull/225